core/kazoo_number_manager/src/knm_port_request.erl

```diff
@@ -332,6 +332,19 @@ transition_numbers(PortReq) ->
             {'error', PortReq}
     end.
 
+-spec create_and_activate(ne_binaries(), knm_number_options:options()) -> knm_numbers:knm_numbers_return().
+create_and_activate(Numbers, Options) ->
+    CreationResults = knm_numbers:create(Numbers, Options),
+    {_OK, Errored} = lists:partition(fun was_number_operation_successul/1, CreationResults),
+    case Errored of
+        [] ->
+            lager:debug("all numbers created, activating numbers"),
+            knm_numbers:move(Numbers, knm_number_options:assign_to(Options), Options);
```

Comment:
@fenollp There was a problem with transition (pending/schedule)->complete ports which I fixed by setting `state` to `available` but it causes the numbers to be in `reserved` state, so here I used `knm_numbers:move` to set the state to `in_service`, just like what `activate` is doing in api.